app/components/Main.js

```diff
@@ -85,13 +86,14 @@ class Main extends React.Component {
     }
   }
 
-  _updateTask() {
+  _updateTask(value, taskEdit) {
```

Comment:
this doesn't need to be in Main anymore, because the socket can emit directly from the child component now.